.github/workflows/preview.yml

```
@@ -0,0 +1,25 @@
+name: Deploy PR previews
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+
+concurrency: preview-${{ github.ref }}
+
+jobs:
+  deploy-preview:
+    runs-on: ubuntu-20.04
```

Why not use `ubuntu-latest` here?
```suggestion
    runs-on: ubuntu-latest
```